aws-py-ecs-instances-autoapi/py-ecs-instance/__main__.py

```diff
@@ -0,0 +1,202 @@
+# Deploys ECS with EC2 container instances (as opposed to using Fargate).
+#
+
+"""An AWS Python Pulumi program"""
+import json
+import pulumi
+import pulumi_aws as aws
+
+aws_config = pulumi.Config("aws")
+region = aws_config.get("region") or "us-east-1" 
+avail_zone = region+"a" # e.g. us-east-1a 
+
+stack_config = pulumi.Config("cfg")
+asg_size = int(stack_config.require("autoscalingGroupSize")) 
+
+# Get the default VPC. 
+default_vpc = aws.ec2.get_vpc(default=True)
+default_vpc_subnets = aws.ec2.get_subnet_ids(vpc_id=default_vpc.id)
+
+# Security group to access the nginx container.
+sg = aws.ec2.SecurityGroup(
+    'nginx-sg',

```

Comment:
Nit: It would be nice if we used a consistent quote or single quote for strings throughout the example.

Reply:
Standardized on double quotes.